mrjob/spark/mrjob_spark_harness.py

```diff
@@ -57,6 +57,12 @@
             'An empty directory to write counter output to. '
             'Can be a path or URI.')
     )),
+    (['--num-reducers'], dict(
+        default=None,
+        dest='num_reducers',
```

Comment:
You should probably add a check in `main()` that ensures that `args.num_reducers` is `None` or a positive number. Setting num reducers to 0 is used as a way to tell Hadoop to skip the reduce step, and we should be explicit that the harness doesn't emulate that.

Reply:
thanks! Didn't aware that!!! I'll update this.